libs/admin-app/profile/src/lib/register-page/register-page.component.ts

```diff
@@ -81,17 +86,30 @@ export class RegisterPageComponent implements OnInit {
         }
         else
         {
-          this.showMessage("Contact number already exists");
+          this.toastService.show('Contact number already exists',{
+            classname: 'bg-danger text-light',
+            delay: 5000,
+            autohide: true
+          });
         }
       }
       else
       {
-        this.showMessage("Email already exists");
+        this.toastService.show('Email already exists',{
+          classname: 'bg-danger text-light',
+          delay: 5000,
+          autohide: true
+        });
```

Comment:
Same here we should probably not  specify if the email or password is incorrect individually.